_posts/en/newsletters/2019-06-12-newsletter.md

```diff
@@ -0,0 +1,294 @@
+    summarizing the tradeoffs and showing the clear overall benefits of
+    Schnorr and Taproot for current multisig spenders.
+
+    In addition to enhancing current uses of Bitcoin, also described are
+    new features that aren't currently practical but which would become
+    so if Schnorr-style signatures and MAST-style script trees become
+    available.  Lee finishes his talk by providing a rough, and heavily
+    caveated, timeline for when we might see the changes described in
+    his talk.
+
+- **Optech celebrates Newsletter #50:** in early June of last year, John
+  Newbery emailed Dave Harding about some plans he had for the newly
+  formed Optech organization---including the single sentence, "We'll
+  also produce some written material [such as] weekly or monthly news
+  summaries."  Being a bit bored that day, Harding replied back with an
+  unsolicited [proof-of-concept newsletter][].  Newbery and other early
+  Optech contributors liked it, so a few details were worked out and
+  regular weekly publication of this newsletter began.
+
+    Fifty issues later, the newsletter now over 2,500 email subscribers
+    and an unknown number of additional unique subscribers via
+    [RSS][newsletter rss], [Twitter][optech twitter], and Max
+    Hillebrand's [readings][hillebrand optech].  We've published over
+    85,000 words overall---about 250 printed pages <!-- 350 words per
+    page --> of content---with draft versions of the newsletters having
+    received a total of 948 comments so far from an amazing team of
+    reviewers who help ensure technical accuracy and readable prose.
+    Announcements of new newsletters have accumulated over 1,300
+    retweets <!-- 1,359 --> and 3,000 likes, <!-- 3,087 --> plus many
+    upvotes on [Reddit][optech reddit].  Most importantly, we've heard
+    directly from many of our readers that they find the newsletter to
+    be especially useful.
+
+    We're amazed, gratified, and humbled that a purely technical
+    newsletter focused solely on Bitcoin has received such an amazing
+    reception in its first year of publication.  We know you all have
+    high expectations from us going forward, and we hope that we can
+    live up to those aspirations in our next fifty issues.  As always,
+    we thank our [founding sponsors][], [member companies][], all the
+    people who have contributed to the newsletter, and everyone who has
+    contributed to Bitcoin development in general---without you, we'd be
+    writing about something much less exciting than the future of money.
+
+## Bech32 sending support
+
+*Week 13 of 24 in a [series][bech32 easy] about allowing the people
+you pay to access all of segwit's benefits.*
+
+{% comment %}<!-- weekly reminder for harding: check Bech32 Adoption
+wiki page for changes -->{% endcomment %}
+
+{% include specials/bech32/13-adoption-speed.md %}
+
+## Notable code and documentation changes
```

Comment:
Is the plan now for this section to cover the previous week (up to friday when you produce the first draft) or do you plan to update it when you give your final draft?

Reply:
So the way I've been doing it for a while is that I have a script I run each day that shows me all the commits I haven't see on each project's master branch and I document the notable changes into a file.  When I start drafting the newsletter, I copy those into the draft.

Then, when I finish the draft a day or two later, I look at how long the newsletter is and decide whether or not to add any of the merges that came in during the intervening time.  Last week and this week we had long newsletters, so I haven't added anything.  In the future we may have short newsletters were I will add anything.

For the devs like you who know when stuff was merged, I suppose that inconsistency around the dating can be annoying (sorry).  For readers, I think it's better to do what we can to try to balance out the slow weeks with the busy weeks.

Reply:
Sounds very reasonable. Thanks for the explanation!